python/ql/test/experimental/library-tests/frameworks/django-v2-v3/routing_test.py

```diff
@@ -36,6 +37,12 @@ def get(self, request, untrusted):  # $ requestHandler routedParameter=untrusted
         return HttpResponse('ClassView get: {}'.format(untrusted))  # $HttpResponse
 
 
+# direct import with full path to `View` class (previously not supported)
```

Comment:
```suggestion
# direct import with full path to `View` class
```
Let us keep the development history out of the comments :-)

Reply:
In general I would agree with this. But in this specific instance, I wanted to highlight why we have both `ClassView` and `ClassView2`. I guess the other part of the comment is enough

Reply:
Merged the suggestion (but kept it open so you can see my response)